user_management/src/user/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls import path
 
-from .views import IsUsernameTakenView, RefreshJWT, SignInView, SignUpView
+from .views import (IsUsernameTakenView, RefreshJWT, SignInView, SignUpView,
+                    UserIdView)
 
 urlpatterns = [
     path('signup/', SignUpView.as_view(), name='signup'),
     path('signin/', SignInView.as_view(), name='signin'),
     path('refresh-jwt/', RefreshJWT.as_view(), name='refresh-jwt'),
     path('username-exist/', IsUsernameTakenView.as_view(), name='username-exist'),
+    path('<str:user_id>/', UserIdView.as_view(), name='user-id')
```

Comment:
change to <int:user_id>

Reply:
I disagree, this is on purpose. In the future we might want to handle things differently and handle string parameters (such as the username)